### DISCUSSION TITLE
选中导出音频功能

### DISCUSSION THREAD

**Discussion by lalalandtuzi** [ZuodaoTech/everyone-can-use-english#630] [General]
选中导出音频功能似乎只能导出某个句子，可以增加更灵活的选段导出功能吗？给开发者递☕

**Comment by an-lee** [comment]
意思是想要将几个句子同时选中，合并一起导出吗？

**Reply by benabcben** [reply]
有计划支持批量导出每一个句子吗，我看现在只能一个一个的导出

**Reply by an-lee** [reply]
可以的。后续加一个。

**Comment by benabcben** [comment]
好的 我的1美金还没返我呢🤣

an-lee ***@***.***> 于2024年7月4日周四 09:17写道：

**Comment by benabcben** [comment]
[image: 747d9ad43dcb3a1bd073e3f420221e5.jpg]
音标的字体颜色太浅了,打印出来看不清了都,下个版本能改下吗

lei xia ***@***.***> 于2024年7月8日周一 16:38写道：
